### PR TITLE
Migrate watcher to SDKv2

### PR DESCRIPTION
**Pull request by panoel**
Hightlights:
- All references to the monorepo are removed from the watcher directory and below
- Code in the monorepo that was needed and not in sdkV2 was copied verbatim
- Node version in .github was bumped from 16 to 20
- Jest was updated due to weird global renaming errors on Ubuntu
- The large number of lines in this PR is due to the package changes.
TIA ❤️

### CODE REVIEW

File: watcher/package.json [hunk 1]
```diff
@@ -36,6 +36,9 @@
     "@types/bn.js": "^5.1.5",
     "@wormhole-foundation/sdk-definitions": "^0.6.2",
     "@wormhole-foundation/sdk-evm": "^0.6.2",
+    "@wormhole-foundation/sdk-evm-core": "^0.6.3",
+    "@wormhole-foundation/sdk-solana": "^0.6.3",
+    "@wormhole-foundation/sdk-solana-core": "^0.6.3",
```
Comment (barnjamin) [inline watcher/package.json:41]: bump all of them or you're gonna have a bad time

Comment (panoel) [reply watcher/package.json:41]: Fixed.

File: watcher/src/databases/BigtableDatabase.ts [hunk 2]
```diff
@@ -237,12 +238,13 @@ export class BigtableDatabase extends Database {
           for (const row of vaaRows) {
             try {
               const vaaBytes = row.data.info.bytes[0].value;
-              const parsed = parseVaa(vaaBytes);
+              // const parsed = parseVaa(vaaBytes);
```
Comment (bingyuyap) [inline watcher/src/databases/BigtableDatabase.ts:241]: Do we still need this commented line? I guess if not we can delete it

File: watcher/src/watchers/NTTSolanaWatcher.ts [hunk 3]
```diff
@@ -155,15 +157,26 @@ export class NTTSolanaWatcher extends SolanaWatcher {
...
+    if (!acctInfo?.data) throw new Error('No data found in message account');
+    const { emitterAddress, sequence, emitterChain, payload } = deserializePostMessage(
+      new Uint8Array(acctInfo.data)
+    );
+    msgData = {
+      sequence,
+      emitterChain,
+      emitterAddress: Buffer.from(emitterAddress.toUint8Array()),
+    };
+    // const postMessage = await getPostedMessage(this.getConnection(), accountKey);
```
Comment (bingyuyap) [inline watcher/src/watchers/NTTSolanaWatcher.ts:174]: seems like this is can be deleted as well

**Review by bingyuyap — APPROVED** [review]
LGTM: 
- identified some undeleted comments that we might want to delete.
- Ran jest on NTTSolanaWatcher as well and it seems to work fine.
- Didn't check NTTUtils since it's copied from elsewhere

File: watcher/src/databases/BigtableDatabase.ts (hunk 2 above)
Comment (panoel) [reply watcher/src/databases/BigtableDatabase.ts:241]: Removed

File: watcher/src/watchers/NTTSolanaWatcher.ts (hunk 3 above)
Comment (panoel) [reply watcher/src/watchers/NTTSolanaWatcher.ts:174]: Removed

File: watcher/src/databases/BigtableDatabase.ts (hunk 2 above)
Comment (panoel) [reply watcher/src/databases/BigtableDatabase.ts]: Removed

File: watcher/src/watchers/NTTSolanaWatcher.ts (hunk 3 above)
Comment (panoel) [reply watcher/src/watchers/NTTSolanaWatcher.ts]: Removed

File: watcher/src/index.ts [hunk 4]
```diff
@@ -23,7 +23,7 @@ const supportedChains: Chain[] =
         // 'Ethereum',
         // 'Terra',
         'Bsc',
-        'Polygon',
+        // 'Polygon',
```
Comment (barnjamin) [inline watcher/src/index.ts:26]: y?

**Review by barnjamin — DISMISSED** [review]

File: watcher/src/index.ts (hunk 4 above)
Comment (panoel) [reply watcher/src/index.ts:26]: Because it is PolygonSepolia, now.

**Review by bingyuyap — APPROVED** [review]

**Pull request merged**